grails-app/conf/application.yml

```diff
@@ -19,21 +19,39 @@ app:
       userId: "X-ALA-userId"
       hostName: "X-ALA-hostname"
 
+auth:
+  baseURL: "https://auth.ala.org.au"
+
+casUrl: "${auth.baseURL}/cas/logout"
+appUrl: "http://devt.ala.org.au:8087/"
 
 security:
-   cas:
-      uriExclusionFilterPattern: ['/assets/.*','/images.*','/css.*','/js.*','/less.*','/ajax/(bulkLookupQuestions|saveBookmarkLocation|getBookmarkLocations)','/uploads/.*']
-      authenticateOnlyIfLoggedInPattern: "/*"
-      uriFilterPattern: ['/user/.*','/site/(?!(index|list|elasticsearch|getImages|getPoiImages|ajaxUpdate|checkSiteName|ajaxList|checkPointInsideProjectAreaAndAddress)).*','/project/(?!(index|search|citizenScience|listRecordImages|projectSummaryReportCallback)).*','/activity/(?!index).*','/output/(?!index).*','/image/(?!index).*','/admin/.*','/proxy/speciesListPost','/proxy/documentUpdate','/proxy/deleteDocument','/home/advanced','/organisation/(?!index).*','/organisation/(?!list).*','/bioActivity/create/.*','/bioActivity/edit/.*','/bioActivity/list','/sight/.*']
-      readOnlyOfficerRole: "ROLE_FC_READ_ONLY"
-      alaAdminRole : "ROLE_ADMIN"
-      officerRole : "ROLE_FC_OFFICER"
-      adminRole : "ROLE_FC_ADMIN"
-#      casServerName: "https://auth.ala.org.au"
-#      casServerLoginUrl: "https://auth.ala.org.au/cas/login"
-#      casServerUrlPrefix: "https://auth.ala.org.au/cas"
-#      logoutUrl: "https://auth.ala.org.au/cas/logout"
-#      loginUrl: "https://auth.ala.org.au/cas/login"
+  cas:
+    enabled: false
+    uriExclusionFilterPattern: ['/assets/.*','/ajax/(saveBookmarkLocation|getBookmarkLocations)','/uploads/.*']
+    authenticateOnlyIfCookieFilterPattern: "/*"
+    uriFilterPattern: ['/user/.*','/site/(?!(index|list|elasticsearch|getImages|getPoiImages|ajaxUpdate|checkSiteName|ajaxList|checkPointInsideProjectAreaAndAddress)).*','/project/(?!(index|search|citizenScience|listRecordImages|projectSummaryReportCallback)).*','/activity/(?!index).*','/output/(?!index).*','/image/(?!index).*','/admin/.*','/proxy/speciesListPost','/proxy/documentUpdate','/proxy/deleteDocument','/home/advanced','/organisation/(?!index).*','/organisation/(?!list).*','/bioActivity/create/.*','/bioActivity/edit/.*','/bioActivity/list','/sight/.*']
```

Comment:
uriFilterPattern does not support regex values. only supports Java Servlet Filter paths.

https://github.com/AtlasOfLivingAustralia/ala-auth-plugin#setup-cas-authentication-for-your-app